config/collector.js

```diff
@@ -56,6 +56,21 @@ const createScanConfig = (dirs, cwdPattern) => ({
             onlyFiles: false,
             onlyDirectories: true
         }
+    },
+
+    twigs: {
```

Comment:
Is this a good key? it is a word that does not exist as such.
Also, it leaks a tool name into what is generically just templates.

As such, I would have assumed/expected the word `templates` to be used, not leaking any of the underlying details.

Reply:
Value of ‘twigs’ key is a collection of views and templates. If I rename key to templates, misunderstanding is possible, because I collect views and templates together. But if ‘twigs’ isn’t ok, I can rename the key to ‘viewsAndTemplates’.

Reply:
Lets separate them at all. So please create separate `templates` and `views` configurations.